### ISSUE TITLE
Calling a plugin from jxcore app.js

### ISSUE THREAD

**Issue by whatevernot** [jxcore/jxcore-cordova#36]
Hi:

I have jxcore working with iOS, but run into one snag and wondering if someone can help.

I am embedding node within an iOS application, and using this as a way to call into the HomeKit APIs and make them available through a REST API (this is all hobby stuff, obviously won't be allowed in app store).

I have node and express running with jxcore inside an iOS app - when an HTTP request comes in, then from the 'app.js' side (in the node server) I would like to call into native code (a plugin) - similar to the web page (which works now).

Basically, inside my app.js on the server I want to do the equivalent of 'cordova.exec'.

Is there a way to 'hook' into the cordova infrastructure and make an invocation from app.js?

When the web side execute 'cordova.exec' I'm unsure how that translates to the native call to the appropriate plug-in.

Any help would be greatly appreciated.

brian...

**Comment by whatevernot** [comment]
Closing this - the workaround is simple, as I can call from jxcore (node) -> cordova (web). Works perfectly. This is a great project - complete lifesaver, cheers.
